database/migrations/20200901222910-CreateTableWallet-Event.js

```diff
@@ -24,13 +22,14 @@ exports.up = function(db) {
       notNull: true,
       defaultValue: new String('now()'),
     },
+    payload: { type: 'json', notNull: true },
```

Comment:
Same issue as above, a new migration should be created to update the table

Reply:
added db-migrate create AddPayloadToWalletEventTable.js